.travis.yml

```diff
@@ -11,18 +11,20 @@ cache:
     - $HOME/google-cloud-sdk
     - $HOME/.m2
 script:
-  - ./mvnw test -B -Dit.spanner=true
-  - ./mvnw test -B -Dit.storage=true
+  - ./mvnw test -B ${INTEGRATION_TEST_FLAGS}
 install:
   - ./mvnw install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
 before_script:
   - gcloud beta emulators pubsub start &
   - while [ ! -f ~/.config/gcloud/emulators/pubsub/env.yaml ]; do sleep 1; done
   - $(gcloud beta emulators pubsub env-init)
 before_install:
-  - openssl aes-256-cbc -K $encrypted_1ef8dfbdb114_key -iv $encrypted_1ef8dfbdb114_iv -in travis.tar.gz.enc -out travis.tar.gz -d
+  - if [ "$encrypted_1ef8dfbdb114_key" != "" ]; then
```

Comment:
So `$encrypted_1ef8dfbdb114_key` is an empty string in PRs from forks? Is that documented anywhere?